build.gradle

```diff
@@ -208,7 +209,11 @@ dependencies {
     provided "appeng:RotaryCraft:${config.rotc.version}:api"
     provided ("appeng:appliedenergistics2:${config.ae2.version}:dev") {
         exclude module: 'buildcraft'
-    }
+    }*/
+
```

Comment:
Oh ok, should I rewrite the whole thing? I'm confused, in the old Version also the whole Mod was used